flink-table-store-flink/flink-table-store-flink-common/src/main/java/org/apache/flink/table/store/connector/FlinkConnectorOptions.java

```diff
@@ -102,7 +102,7 @@ public class FlinkConnectorOptions {
     public static final ConfigOption<Duration> CHANGELOG_PRODUCER_FULL_COMPACTION_TRIGGER_INTERVAL =
             key("changelog-producer.compaction-interval")
                     .durationType()
-                    .defaultValue(Duration.ofMinutes(30))
+                    .defaultValue(Duration.ofSeconds(0))
```

Comment:
Could `changelog-producer.compaction-interval` has no default value? If the value of `changelog-producer.compaction-interval` is null or 0, the default behavior is both that each checkpoint has a full compression and generates a change log.

Reply:
Yes, change to default to a full compression for each ck.

Reply:
@xuzhiwen1255, therefore, this changes to `noDefaultValue()`?

Reply:
If there's no default, isn't that a null pointer exception

Reply:
noDefaultValue will cause NullPointerException,so the default is 0